File: 1442-count-triplets-that-can-form-two-arrays-of-equal-xor/1442-count-triplets-that-can-form-two-arrays-of-equal-xor.js
/**
 * @param {number[]} arr
 * @return {number}
 */
var countTriplets = function(arr) {
    let xors = 0;
    for(let i = 0; i < arr.length; i++){
        let cur = arr[i];
        for(let j = i+1; j < arr.length; j++){
            cur ^= arr[j]
            if(cur == 0){
                xors += j - i
            }
        }
    }
    return xors
};

File: 1442-count-triplets-that-can-form-two-arrays-of-equal-xor/1442-count-triplets-that-can-form-two-arrays-of-equal-xor.py
class Solution:
    def countTriplets(self, arr: List[int]) -> int:
        count = 0
        for i in range(0,len(arr)):
            xor = arr[i]
            for j in range(i+1, len(arr)):
                xor ^= arr[j]
                if xor == 0:
                    count += (j - i)
        return count